content/app/development/ux/pages/_index.en.md

```diff
@@ -62,4 +62,21 @@ The available settings are the following:
 | order                 | Array   | see [Navigation](/app/development/ux/pages/navigation/#order)                                                                                                                  |
 | excludeFromPdf        | Array   | see [PDF](/app/development/ux/pdf/#automatic-configuration)                                                                                                                    |
 
+## Expanded width
+
+You can set the default value of a page to be expanded by adding the `expandedWidth` property to a layout in the `data`
```

Comment:
This wording is a bit though to read and understand. 
Would this be a formulation that still makes sense?

To make a page automatically expand to the full width of the browser window when opened, you can add the expandedWidth property to a layout within the data property. This sets the default state of the page to be expanded.